classes/class-wc-rest-connect-stripe-account-controller.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+if ( class_exists( 'WC_REST_Connect_Stripe_Account_Controller' ) ) {
+	return;
+}
+
+class WC_REST_Connect_Stripe_Account_Controller extends WC_REST_Connect_Base_Controller {
+	protected $rest_base = 'connect/stripe/account';
+
+	public function post( $request ) {
+		$data = $request->get_json_params();
+
+		// Whitelist the properties we want.
+		$body = array(
+			'email' => $data['email'],
+			'country' => $data['country'],
```

Comment:
Would you mind adding a comment here or somewhere that country should be an " ISO 3166-1 alpha-2 country code" per https://stripe.com/docs/api/node#create_account